message-schemas/generated/proto/foxglove/builtins.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
```

Comment:
What makes these "builtins" vs separate Time.proto and Duration.proto files?

Reply:
Perhaps nothing